src/js/components/Word.js

```diff
@@ -23,11 +23,15 @@ export function Word(props, ...children) {
             }
           },
           onKeydown: e => {
-            const key = e.key || e.keyCode;
-            if (key === 'Enter' || key === 13) {
+            const { key } = e;
+            const { keyCode } = e;
```

Comment:
You could destructure these on the same line (as well as the target),
```js
const { key, keyCode, target } = e;
```

Reply:
Great suggestion; I've implemented those changes and it looks like it's running smoothly.